### DISCUSSION TITLE
How do I figure out which version of Helm my argocd deployment is using?

### DISCUSSION THREAD

**Discussion by igorpalant** [argoproj/argo-cd#5671] [Q&A]
I have an issue deploying dapr 1.0.0. Here is the issue and I am wondering if it is because of certain helm version:

 failed exit status 1: Error: parse error in "dapr/charts/dapr_sidecar_injector/templates/dapr_sidecar_injector_webhook_config.yaml": template: dapr/charts/dapr_sidecar_injector/templates/dapr_sidecar_injector_webhook_config.yaml:1: function "lookup" not defined

**Comment by jgwest** [comment]
Hi @igorpalant, if by version you mean the specific version of the Helm utility, then here are a couple ways:
- Check the [tools-versions.sh](https://github.com/argoproj/argo-cd/blob/master/hack/tool-versions.sh) file, which is the list of tool versions that are packaged into the Argo CD repo-server image. Ensure that you select the branch (for example, `release-1.8`) or tag that corresponds to the release you are using.
- Or, if you are able to access the Deployments of your Argo CD, you can shell into the repo-server:
    - Example: `kubectl exec -it pod/example-argocd-repo-server-7c955bb447-k67jb bash`, then `helm2 version` and `helm version`

If by version you mean Helm API version (v2 or v3), then see [here](https://argoproj.github.io/argo-cd/user-guide/helm/#helm-version) for the information on which version is used and why.
